lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -732,6 +732,306 @@ class DecomposeAtenTriuOp : public OpRewritePattern<AtenTriuOp> {
 };
 } // namespace
 
+Value arange(PatternRewriter &rewriter, Location loc, Value size, Value dtype,
```

Comment:
nit: I don't think this is needed as a helper function since there is not much going on other than creating the `AtenArangeOp`. I think it is better to keep the `rewriter.create<AtenArangeOp>` explicit every time it is needed in the code